### PR TITLE
Fixed minor errors for ELD cards

### PR DESCRIPTION
**Pull request by jmharmon**

### CODE REVIEW

**Comment by jmharmon** [comment]
Before this fix.
Screen Shot 2019-09-04 at 8 48 29 PM

File: Mage.Sets/src/mage/cards/c/CrystalSlipper.java [hunk 1]
```diff
@@ -27,10 +27,8 @@ public CrystalSlipper(UUID ownerId, CardSetInfo setInfo) {
         this.subtype.add(SubType.EQUIPMENT);
 
         // Equipped creature gets +1/+0 and has haste.
-        Ability ability = new SimpleStaticAbility(new BoostEquippedEffect(1, 0));
-        ability.addEffect(new GainAbilityAttachedEffect(
-                HasteAbility.getInstance(), AttachmentType.EQUIPMENT).setText("and has haste")
-        );
```
Comment (theelk801) [inline Mage.Sets/src/mage/cards/c/CrystalSlipper.java]: I changed this to fix the text, I just forgot to add `this.addAbility(ability);`

Comment (jmharmon) [reply Mage.Sets/src/mage/cards/c/CrystalSlipper.java]: Changed it back and added the missing part.

Comment (theelk801) [reply Mage.Sets/src/mage/cards/c/CrystalSlipper.java]: Great, thanks for your contributions

**Pull request merged**